test/data/testSP.py

```diff
@@ -68,11 +68,40 @@ def testatomnos(self):
     @skipForLogfile('Molpro/basicMolpro2006', "These tests were run a long time ago and since we don't have access to Molpro 2006 anymore, we can skip this test (it is tested in 2012)")
     @skipForParser('Turbomole','The parser is still being developed so we skip this test')
     def testatomcharges(self):

```

Comment:
Should this then have a Mulliken suffix?

Reply:
This was intentional, to have `testatomcharges` be generic for all types and `testatomcharges_mulliken` and `testatomcharges_lowdin` to make sure Mulliken and Lowdin are present. Is there a better way of doing this? I can at least improve the docstring to something like `Are any present atomic charges consistent with natom?`.

Reply:
The change in description helped this make sense to me. I see why both the generic and specific tests are needed.